### PR TITLE
Suppress password output

### PR DESCRIPTION
**Pull request by vjagaro**
This will suppress the master password from being displayed to the terminal.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -32,16 +34,15 @@ func main() {
 	if err != nil {
 		errAndExit("Improperly formatted JSON file: %v", err)
 	}
-	fmt.Println("Please input your master password")
-	var reader = bufio.NewReader(os.Stdin)
-	password, err := reader.ReadBytes('\n')
+	fmt.Print("Please input your master password: ")
+	password, err := term.ReadPassword(int(syscall.Stdin))
+	fmt.Println()
```
Comment (graynk) [inline main.go]: A really minor thing, but could you remove this `Println` and make the above `Print` into `Println` instead?

File: main.go [hunk 2]
```diff
@@ -32,16 +34,15 @@ func main() {
 	if err != nil {
 		errAndExit("Improperly formatted JSON file: %v", err)
 	}
-	fmt.Println("Please input your master password")
-	var reader = bufio.NewReader(os.Stdin)
-	password, err := reader.ReadBytes('\n')
+	fmt.Print("Please input your master password: ")
+	password, err := term.ReadPassword(int(syscall.Stdin))
```
Comment (graynk) [inline main.go]: Explicit conversion to `int` is redundant here, I believe

**Review by graynk — CHANGES_REQUESTED** [review]
A couple of small notes

File: main.go (hunk 1 above)
Comment (vjagaro) [reply main.go]: No problem.

File: main.go (hunk 2 above)
Comment (vjagaro) [reply main.go]: Good catch.

Comment (graynk) [reply main.go]: Turns out explicit cast to `int` _was_ necessary, so sorry for that :)
Apparently, on Windows `syscall.Stdin` changes type from `int` to `syscall.Handle`, which is an alias for `uintptr`: https://pkg.go.dev/syscall?GOOS=windows#pkg-variables

**Pull request merged**